src/fullStoryWrapper.js

```diff
@@ -35,12 +35,12 @@ class FSWrapper {
   }
 
   async tryIdentifySession (profile, reportingUsersBlacklist) {
-    if (!profile || !profile.user_id) {
-      this._warn('A profile is required to identify this session.');
-      return false;
+    const sProfile = profile || {};
+    if (!sProfile.user_id) {
```

Comment:
This still assumes that the profile has user_id to identify the user.

Instead, we can have `tryIdentifySession(user, profile, ...)`

Reply:
It is no longer required. the session will be reported as anonymous.

Reply:
It will "silently" not do what the user of this package wants. I feel we should limit the chances for this.

Reply:
Your request is a breaking change. We would have to create a new major version of the package.